config/hub/rbac/role.yaml

```diff
@@ -11,6 +11,17 @@ rules:
   verbs:
   - list
   - watch
+- apiGroups:
+  - ""
+  resources:
+  - namespaces
+  verbs:
+  - create
```

Comment:
We don't need list, patch, or watch, right?

Reply:
We don't. I can remove it if you feel it is better to have fewer permissions.

Reply:
Yeah, remove them if not needed.

Reply:
Done.

Reply:
@BenamarMk I had to add list and watch again as controller-runtime requires list and watch permissions for all resources that we do a get on. It is because of the client side caching that it sets up automatically for all the resource types we do a get on.